src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -363,6 +368,12 @@ private static bool GetBooleanQueryParameterValue(NameValueCollection queryStrin
             return bool.TryParse(value, out bool parsedValue) && parsedValue;
         }
 
+        private static bool? GetNullableBooleanQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName)

```

Comment:
This would be a slightly more natural and simpler way to design this API.
```csharp
private static bool GetBooleanQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName, bool defaultValue)
{
    var value = queryStringNameValueCollection[queryParameterName];
    return bool.TryParse(value, out bool parsedValue) ? parsedValue : defaultValue;
}
```